src/pages/workspaces/workspace/Analyses.js

```diff
@@ -8,8 +8,8 @@ import { requesterPaysWrapper, withRequesterPaysHandler } from 'src/components/b
 import { withViewToggle } from 'src/components/CardsListToggle'
 import {
```

Comment:
Could you put these on one line. An import per line takes up a lot of vertical space in the file. Could you do the same on line 21?

Reply:
ah my IDE likes doing this